common/src/main/java/net/william278/huskhomes/redis/redisdata/RedisAbstract.java

```diff
@@ -0,0 +1,68 @@
+package net.william278.huskhomes.redis.redisdata;
```

Comment:
I'm not keen on this package name (redisdata) -- could we rename it to "lettuce" or something?